src/utils/networkConfig.ts

```diff
@@ -1,4 +1,34 @@
-type MinimalNetworkConfig = { rpcAddrs: string[]; chainName: string };
+type NetworkNotice = {
+  start: string;
+  // In the future this might be optional to indicate that it's user-dismissable.
+  // In that case the client would need some persistent state, perhaps keyed by `message`.
+  end: string;
+  message: string;
+};
+
+type MinimalNetworkConfig = {
+  rpcAddrs: string[];
+  chainName: string;
+  notices?: NetworkNotice[];
```

Comment:
Do we want to have separate notices for PSM and Vaults?

Reply:
if we do I think that has to go somewhere else. this is config for the network. appropriate for things like pending upgrades.

Reply:
Makes sense